Makefile

```diff
@@ -1,3 +1,4 @@
+UNAME_S := $(shell uname -s | tr '[:upper:]' '[:lower:]')
```

Comment:
Instead of using `uname`, this should use `go env` as:

```
GO_HOST_OS := $(shell go env GOHOSTOS)
```

Reply:
yeap. done